models/datasets/generic/ds__lab_requests.yml

```
@@ -0,0 +1,120 @@
+version: 2
+
+models:
+  - name: ds__lab_requests
+    description: Lab Requests
```

Sentence casing for consistency?
```suggestion
    description: Lab requests
```